python/tk_3dsmaxplus/maxscript.py

```diff
@@ -51,6 +51,13 @@ def create_menu(menu_name, menu_var):
         '''.format(menu_var=menu_var, menu_name=menu_name))
 
     @staticmethod
+    def destroy_menu(menu_name):
+        MaxPlus.Core.EvalMAXScript('''
+            menu = menuMan.findMenu "{menu_name}"
+            if menu != undefined then menuMan.unregisterMenu menu
```

Comment:
I found out that we can use MaxPlus for menu management instead of Maxscript, so I will use MaxPlus.MenuManager.UnregisterMenu(MENU_LABEL) in all places and remove destroy_menu() in maxscript.py